hearts/game/manager.py

```diff
@@ -612,6 +645,9 @@ def get_game_state(self, player: Player, is_observer: bool = False) -> dict:
                     }
                 game_state['trick'][relative_pos] = card
 
+        if self.game.winning_player:
+            game_state['trick'] = {}
```

Comment:
Just clearing the trick object if the game is over. This just means the game screen goes to an empty state when it's over.